resources/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Self Paced Learning Material
+---
+
+There is a lot of learning resources on the internet, it can be overwhelming to figure out will work best for you and where to start. The following are resources that Pyladies PDX members have found useful to them, depending on background.
+
+## Tutorials for Beginners, new to programming
+
+If you're brand new, you'll first need to get your computer set up to start programming. We recommend the [Django Girls Installation Instructions](https://tutorial.djangogirls.org/en/installation/#installation). These instructions are great regardless of if you're interested in Django (a web framework). The first paragraph on this page says you can skip if reading from home... don't skip it, the whole point is to get your computer set up. Follow each section up until the "What is Django" chapter. These instructions cover Mac, Linux and Windows, and are frequently updated.
+
+Once you've got everything installed, and know a little bit about the command line, you're ready to dive into Python. Depending on why you're interested in learning python, we recommend the following tutorials:
+
+### Web development:
+
+There are a lot of web frameworks in Python. If you don't already know which framework you need, we recommend the excellent [Django Girls tutorial](https://tutorial.djangogirls.org/en/).
+
+### Science/Data Analysis:
+
+[Software Carpentry](https://swcarpentry.github.io/python-novice-inflammation/) Programming skills for science research
+
+[Data Carpentry](http://www.datacarpentry.org/python-ecology-lesson/) Focus on data organization, cleanup, analysis, and visualization
+
+### Generic Beginner Python
+
+[Think Python](http://greenteapress.com/wp/think-python/) TODO: description (Jules)
```

Comment:
Address this TODO?

Reply:
Free and highly recommended introduction to Python that also touches on general programming and computer science concepts as well. Each chapter includes both straight-forward and challenging exercises. Leans a bit academic and math oriented, so it's especially good if you're comfortable with high-school/early college level math problems, and ready to start learning by doing. If you're looking for a gentle introduction, try another resource first and come back for this one later.

Reply:
Let me know if that' too long and I'll try to shave it down.